service/routes.go

```diff
@@ -89,6 +89,15 @@ func getSchemas() *client.Schemas {
 	authconfig.ResourceMethods = []string{"GET", "POST"}
 	authconfig.PluralName = "configs"
 
+	// TestAuthConfig
+	testAuthconfig := schemas.AddType("testAuthConfig", model.TestAuthConfig{})
+	testAuthconfig.CollectionMethods = []string{"GET", "POST"}
+	testAuthconfig.ResourceMethods = []string{"GET", "POST"}
```

Comment:
Resource methods are the available methods for a specific resource at `/v1-auth/testlogin/<id>`
Does that path even exist/resolve?